src/extension/.vscode/launch.json

```diff
@@ -10,6 +10,7 @@
       "type": "extensionHost",
       "request": "launch",
       "runtimeExecutable": "${execPath}",
+      "env": {"DEBUG": "yes"},
```

Comment:
Do you think it makes more sense for this one to be a boolean variable? (i.e. DEBUG=1 for debug mode, or DEBUG=0 or unset for production mode)?

Reply:
Honestly it would make sense if numbers were allowed since 1 and 0 work like a boolean. In fact originally i tried doing true or false. But it said it had to be a string or null so I chose "yes" here. Do you think "1" would make more sense?

Reply:
Im up for suggestions on different string possibilities as well such as "on", "enabled" etc

Reply:
Now that I think about it "1" == true in javascript/typescript, also non null values also equal true in an if statement, do you think I could just set the "DEBUG": "1" here and then set the if to
```if (process.env.DEBUG) { .... } else {...}```?

Reply:
Oh now that you mention the type limitation there I think I'm okay either way. A different approach could be `NODE_ENV: "dev"` instead.

Reply:
Done!